.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Clone govuk-content-schemas
+        uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: tmp/govuk-content-schemas
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/cache@v1
+        with:
+          path: tmp/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - run: |
+          bundle config path tmp/bundle
+          bundle install --jobs 4 --retry 3
+      - run: bundle exec rubocop
+      - run: bundle exec rake assets:precompile
+      - run: bundle exec rake
```

Comment:
Not for now, necessarily, but have we considered running RuboCop as part of the default Rake task?